src/libstd/rt/unwind.rs

```diff
@@ -160,7 +160,7 @@ pub fn panicking() -> bool {
 // An uninlined, unmangled function upon which to slap yer breakpoints
 #[inline(never)]
 #[no_mangle]
-fn rust_panic(cause: Box<Any + Send>) -> ! {
+pub fn rust_panic(cause: Box<Any + Send>) -> ! {
```

Comment:
This is why I'm so confused, it's not at all clear to me how this works, since all evidence suggests that this symbol should not be exported, nonetheless setting a breakpoint on it DTRT